console/src/app/pages/projects/apps/app-create/app-create.component.ts

```diff
@@ -328,6 +331,11 @@ export class AppCreateComponent implements OnInit, OnDestroy {
     }
   }
 
+  public onChangeDevMode(ob: MatLegacySlideToggleChange) {
+    this.devMode = ob.checked;
+    this.oidcAppRequest.setDevMode(this.devMode);
+  }
+
```

Comment:
```suggestion
```

remove this if we use `ngModel`

Reply:
Thanks @peintnermax for you review as always and sorry if I make silly mistakes, my trip from React to Angular sometimes has issues :smile: 

I see no problem about using ngModel to keep things clean, but where do you suggest I should add 

`this.oidcAppRequest.setDevMode(this.devMode)`

so the devMode value is added to the request and sent when the new app is created. Should I add a set/get instead?

Reply:
no problem. thx for your effort as always.. 👍 
I did totally oversee that sorry. We also use redirectUris as ngModel but define the setter and getter classes which directly set the value to our `oidcAppRequest`

```
public get devMode() {
    return this.oidcAppRequest.toObject().devMode;
  }

  public set devMode(value: boolean) {
    this.oidcAppRequest.setDevMode(value);
  }
```

then we can also remove the global variable

Reply:
Perfect @peintnermax , that will be the approach then, give me some minutes and I'll fix the code and do some testing